docs/en/observability/apm/collect-application-data/index.asciidoc

```diff
@@ -0,0 +1,101 @@
+[[apm-collect-application-data]]
+= Collect application data
+
+[discrete]
+== Language-specific options
+
+Use Elastic APM agents or an OpenTelemetry language SDK to instrument a service in the language its written in:
+
+* <<apm-agents,*Elastic APM agents*>>: Elastic APM agents are instrumentation libraries written in the same language as your service.
+* <<apm-open-telemetry,*OpenTelemetry*>>: OpenTelemetry is an open source set of APIs, SDKs, tooling, and integrations that enable the capture and management of telemetry data from your services and applications.
+** This option includes Elastic Distributions of OpenTelemetry, which are customized versions of https://opentelemetry.io/docs/languages/[OpenTelemetry language SDKs] that are optimized to work with an Elastic backend.
+
+*Not sure which method is right for you?* Compare the available options below.
+
+[discrete]
+=== Capabilities
+
+////
+TO DO:
+Is this table even helpful?
+Is it missing anything that would make it more helpful?
+////
+|===
+| | Elastic APM agents | Elastic Distribution of OpenTelemetry
+
+| *Support level*
+| Fully supported
+| Mixed support +
+(refer to the <<apm-collect-data-availability,availability table>>)
+
+| *Data protocol*
+// TO DO: What is this called?
+| ??
+| https://opentelemetry.io/docs/specs/otel/protocol/[OpenTelemetry protocol (OTLP)]
+
+| *Central configuration*
+| Supported +
+(refer to <<apm-agent-configuration,APM agent central configuration>>)
+| Not supported
+|===
+
+[discrete]
+[[apm-collect-data-availability]]
+=== Availability
+
+////
+TO DO:
+Are we ok with maintaining this table?
+Are we ok with the gaps it exposes?
+////
+
+:available: image:images/icons/check.svg[Available]
+:available-in-preview: image:images/icons/beaker.svg[Available in preview]
+:not-available: image:images/icons/cross.svg[Not available]
+
+*Key*
+
+:hardbreaks-option:
+
+{available} Available
+{available-in-preview} Available in preview
+{not-available} Not available
+
+:!hardbreaks-option:
+
+[cols="<,^,^"]
+|===
+| *Language* | *Elastic APM agent* | *Elastic Distributions of OpenTelemetry*
+| Android | {available} | {not-available}
+| Go | {available} | {not-available}
+| iOS | {available} | {not-available}
+| Java | {available} | {available}
+| .NET | {available} | {available-in-preview}
+| Node.js | {available} | {available-in-preview}
+| PHP | {available} | {available-in-preview}
+| Python | {available} | {available-in-preview}
+| Ruby | {available} | {not-available}
+|===
+
+////
+TO DO:
+Does this framing make sense?
+////
+[discrete]
+== Service-specific options
+
+Elastic also offers several tools to help you collect data from specific services:
+
+* <<apm-k8s-attacher,*Kubernetes*>>: The Elastic APM attacher for Kubernetes simplifies the instrumentation and configuration of your application pods.
+* <<apm-monitoring-aws-lambda,*AWS Lambda Functions*>>: Helps you monitor your AWS Lambda functions.
```

Comment:
Can these link to the end destination instead of the stub pages? It would save a click.

Reply:
I think so. Does that seem confusing to you?